crates/amaru-stores/src/rocksdb/mod.rs

```diff
@@ -423,7 +423,7 @@ impl Store for RocksDB {
                 dreps::tick(&batch, voting_dreps, {
                     let slot = point.slot_or_default();
                     self.era_history
-                        .slot_to_epoch(slot)
+                        .slot_to_epoch(slot.into())
```

Comment:
This is why I don’t like `impl From` for these types: on github it is quite unclear what the conversion is here.

(I know what it does in this case, the point I want to make is more general)